test/test_util.py

```diff
@@ -531,3 +533,10 @@ def test_generate_grid(self):
         # Check 1st and last generated configurations completely:
         self.assertEqual(generated_grid[0].get_dictionary()['float1'], -1.0)
         self.assertEqual(generated_grid[-1].get_dictionary()['float1'], 1.0)
+
+    def test_hyperparameter_json_serialization(self):
```

Comment:
I think this is the wrong file for such a test and it should rather go into `test/read_and_write/test_json.py`.

Reply:
Done.